source/extensions/filters/network/thrift_proxy/conn_manager.h

```diff
@@ -127,6 +127,7 @@ class ConnectionManager : public Network::ReadFilter,
     // ThriftFilters::DecoderFilterCallbacks
     uint64_t streamId() const override { return parent_.stream_id_; }
     const Network::Connection* connection() const override { return parent_.connection(); }
+    Event::Dispatcher& dispatcher() override { return parent_.dispatcher(); }
```

Comment:
nit: wondering if all we need is `TimeSource& timeSource() { return parent_.dispatcher().timeSource(); }`

Reply:
for other filter interfaces in other protocols, we expose the entire `dispatcher`, so its more consistent this way